src/aws/osml/tile_server/viewpoint/__init__.py

```diff
@@ -1,15 +1,19 @@
 #  Copyright 2023-2024 Amazon.com, Inc. or its affiliates.
 
 # flake8: noqa
-from .database import DecimalEncoder, ViewpointStatusTable
-from .models import (
-    ViewpointApiNames,
-    ViewpointListResponse,
-    ViewpointModel,
-    ViewpointRequest,
-    ViewpointStatus,
-    ViewpointUpdate,
+
+from .router import viewpoint_router
+from .viewpoint_id import (
```

Comment:
Does the viewpoint package need to expose all these functions directly or should it really just expose the one viewpoint_router that it is mapping all these calls to the API?

Seems like this is something that can be checked for each __init__.py in the new hierarchy.

Reply:
Agree, cleaned up all the __init__.py files.